### PR TITLE
oxygen: init oxygen

### PR DESCRIPTION
**Pull request by Sanek-creed**

### CODE REVIEW

File: oxygen.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Monthly",
+			"forum": "https://forum.xda-developers.com/t/rom-13-0-unofficial-oxygen-crdroid-v9-1-24-01-2023.4545371/",
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://twrp.me/xiaomi/xiaomimimax2.html",
+			"paypal": "",
+			"telegram": "https://t.me/MiMax_2",
+			"dt": "Sanek-creed/android_device_xiaomi_oxygen",
```
Comment (gwolf2u) [inline oxygen.json]: use crDroid fork url please

File: oxygen.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "",
+			"kernel": "https://github.com/Sanek-creed/kernel_xiaomi_oxygen"
```
Comment (gwolf2u) [inline oxygen.json]: same here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by Sanek-creed** [comment]
Done

File: oxygen.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://twrp.me/xiaomi/xiaomimimax2.html",
+			"paypal": "",
+			"telegram": "https://t.me/MiMax_2",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_oxygen",
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/kernel_xiaomi_oxygen"
```
Comment (gwolf2u) [inline oxygen.json]: Not found
Missing "android_"

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**